### PR TITLE
Explain the need for a build platform security model

### PR DESCRIPTION
**Pull request by arewm**
Addresses #706

### CODE REVIEW

**Review by trishankatdatadog — APPROVED** [review]
LGTM

**Review by abacchi — APPROVED** [review]
Looks good

**Review by MarkLodato — APPROVED** [review]

**Review by lehors — APPROVED** [review]
LGTM

**Review by joshuagl — APPROVED** [review]

**Review by lehors — APPROVED** [review]

File: docs/spec/v1.0/verifying-systems.md [hunk 1]
```diff
@@ -78,7 +78,12 @@ and outputs.
 ![image](build-model.svg)
 
 The following sections detail these elements of the build model and give prompts
-for assessing a build system's ability to produce SLSA Build L3 provenance.
+for assessing a build system's ability to produce SLSA Build L3 provenance. The
```
Comment (lehors) [inline docs/spec/v1.0/verifying-systems.md:81]: Shouldn't that be:
```suggestion
for assessing a build platform's ability to produce SLSA Build L3 provenance. The
```

Comment (arewm) [reply docs/spec/v1.0/verifying-systems.md:81]: That is changed here: https://github.com/slsa-framework/slsa/pull/810/files#diff-7fcdb82839b4a970dcef073ff8bb736ac644fe2d95e4f2ecde2bad6b374420b9R81

I suspect I will have to resolve the merge conflict for editing so close to each other.

**Pull request merged**